Source/Tools/FEXLoader/LinuxSyscalls/x32/IoctlEmulation.cpp

```diff
@@ -600,7 +606,8 @@ namespace FEX::HLE::x32 {
           FDToHandler.SetFDHandler(fd, Virtio_Handler);
         }
         else {
-          LogMan::Msg::EFmt("Unknown DRM device: '{}'", Version.name);
+          LogMan::Msg::IFmt("Unknown DRM device: '{}'", Version.name);
```

Comment:
Perhaps expand the log like:

```suggestion
          LogMan::Msg::IFmt("Unknown DRM device: '{}'. Using default passthrough", Version.name);
```

just to make the passthrough behavior explicit in the log

Reply:
Changed over to it.